dspace_import.py

```diff
@@ -172,79 +175,99 @@ def import_licenses():
     Import data into database.
     Mapped tables: license_label, extended_mapping, license_definitions
     """
-    global eperson_id
-
+    global eperson_id, statistics
+    imported = 0
     # import license_label
     json_a = read_json('license_label.json')
-    if json_a:
-        for i in json_a:
-            json_p = {'label': i['label'], 'title': i['title'], 'extended': i['is_extended'], 'icon': None}
-            # find image with label name
-            try:
-                image_path = migration_const.ICON_PATH + i['label'].lower() + ".png"
-                if os.path.exists(image_path):
-                    with open(image_path, "rb") as image:
-                        file = image.read()
-                        json_p['icon'] = list(file)
-            except Exception as e:
-                log("Exception while reading label image with name: " + i['label'].lower() + ".png occurred: " + e)
-            try:
-                response = do_api_post('core/clarinlicenselabels', None, json_p)
-                created_label = convert_response_to_json(response)
-                del created_label['license']
-                del created_label['_links']
-                labels[i['label_id']] = created_label
-            except:
-                log('POST request ' + response.url + ' failed. Status code ' + str(response.status_code))
+    if not json_a:
+        logging.info("License_label JSON is empty.")
+        return
+    for i in json_a:
+        json_p = {'label': i['label'], 'title': i['title'], 'extended': i['is_extended'], 'icon': None}
+        # find image with label name
+        try:
+            image_path = migration_const.ICON_PATH + i['label'].lower() + ".png"
+            if os.path.exists(image_path):
+                with open(image_path, "rb") as image:
+                    file = image.read()
+                    json_p['icon'] = list(file)
```

Comment:
This if could be refactored: if the path doesn't exist do nothing

Reply:
I did it, because if the path doesn't exist, we still want to run code belowe (201). And if we will use doesn't exist, there will be two lines more. Do you think, I should use not exist -> pass, else: ... ? I think, it is not necessary.